edx_proctoring/static/proctoring/js/proctored_app.js

```diff
@@ -5,4 +5,16 @@ $(function() {
         model: new ProctoredExamModel()
     });
     proctored_exam_view.render();
+
+    // show the timer bar when the window scrolls.
+    $(window).scroll(function() {
+        if ($(this).scrollTop() > 60){
```

Comment:
Is there a way to compute this rather than hard coding "60" here. My sense is that this won't be correct if the page header region changes in height.

Sorry, also just noticed that this scroll event handler is in the proctored_app.js file. Seems like it should be in the proctored_exam_view only if the timer bar is visible.

I think we only have to compute this once per page load, so if we can compute it, we should store it.